go/adbc/pkg/gen/main.go

```diff
@@ -0,0 +1,184 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package main
+
+import (
+	"bytes"
+	"errors"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+	"text/template"
+
+	"golang.org/x/tools/go/packages"
+)
+
+const Ext = ".tmpl"
+
+func formatSource(in []byte) ([]byte, error) {
+	r := bytes.NewReader(in)
+	cmd := exec.Command("goimports")
+	cmd.Stdin = r
+	out, err := cmd.Output()
+	if err != nil {
+		var ee *exec.ExitError
+		if errors.As(err, &ee) {
+			return nil, fmt.Errorf("error running goimports: %s", string(ee.Stderr))
+		}
+		return nil, fmt.Errorf("error running goimports: %s", string(out))
+	}
+
+	return out, nil
+}
+
+func formatCSource(in []byte) ([]byte, error) {
```

Comment:
pre-commit would just take care of this for you, though I guess it requires a little setup.

Reply:
If you have it locally, `pre-commit run [--all-files]` will fix up the files for you, then you can stage and commit